Dockerfile

```diff
@@ -19,7 +19,7 @@ WORKDIR /app
 
 # run crond as main process of container
 ENTRYPOINT ["/bin/sh", "/app/entrypoint.sh"]
-CMD ["dishStatusInflux.py"]
+CMD ["dishStatusInflux.py", "-t", "30"]
```

Comment:
I don't know if you care about this, but I changed the default behavior of this script to match the other ones. This change will maintain the default for the Docker image to the old script default behavior. If you don't think that's important, then I can revert this part.

Reply:
I think this may cause a conflict with the passing of same parameter via ENV. My thought was order of preference would be CLI-passed, ENV, Default when specified so I think the addition of this would always override its ENV equivalent, which would be undesired.

Reply:
Hmm... good point. I guess I'd rather not be doing this, anyway. Any objection to just having the default when run through docker to be not to loop (same as not running through docker)?

Reply:
OK, I have pulled that part of the change. There are now no changes to Dockerfile in this merge.